pkg/trace/pb/stats.proto

```diff
@@ -15,38 +15,45 @@ message StatsPayload {
 	bool clientComputed = 5;
 }
 
-// TODO(gbbr): doc
+// ClientStatsPayload is the first layer of span stats aggregation. It is also
+// the payload sent by tracers to the agent when stats in tracer are enabled.
 message ClientStatsPayload {
-	string hostname = 1;
-	string env = 2;
-	string version = 3;
+        // hostname is the tracer hostname. It's extracted from spans with "_dd.hostname" meta
+        // or set by tracer stats payload when hostname reporting is enabled.
+	string hostname = 1; // env tag set on spans or in the tracers, used for aggregation
+	string env = 2; // env tag set on spans or in the tracers, used for aggregation
+	string version = 3; // version tag set on spans or in the tracers, used for aggregation
 	repeated ClientStatsBucket stats = 4 [(gogoproto.nullable) = false];
-	string lang = 5;
-	string tracerVersion = 6;
-	string runtimeID = 7;
-	uint64 sequence = 8;
+	string lang = 5; // informative field not used for aggregation
+	string tracerVersion = 6; // informative field not used for aggregation
```

Comment:
We should write what they _are_ not where they are used or where they are set. What they specify. What their values represent :)